.github/workflows/linux.yml

```diff
@@ -139,3 +139,38 @@ jobs:
     - name: Compile and test
       run: |
         docker exec CI_container /bin/bash -c "./Package/.github/scripts/compile_and_test.sh"
+
+
+  dev:
+    runs-on: ubuntu-18.04
```

Comment:
I think this should get `continue-on-error:` but sadly [not fully correctly implemented](https://github.community/t/continue-on-error-allow-failure-ui-indication/16773/12)

Reply:
let's just not make the dev builds required. then we see that they fail and either ignore or not. Otherwise we will only notice too late.